_posts/2015/2016-07-20-fullstackcon-2016-retrospective.md

```diff
@@ -0,0 +1,49 @@
+In this post, I want to share my key takeaways from three very intense conference days.
+
+## 1. JavaScript is spreading out
+
+{% imagecaption blog/blog-tessel.png 30% %}
+An LED sound level meter built within a few hours of JavaScript hacking.
+{% endimagecaption %}
+
+With JavaScript rapidly becoming a language to program the **Internet of Things (IoT)**, quite a few talks at the conference covered topics like remote controlling [bluetooth][bluetooth-talk] and [medical devices][pacemaker-talk], or [Virtual Reality][vr-talk]. In one of the workshops around minicomputers and microcontrollers, my colleague Erik hacked an LED sound level meter using Node.js on the [Tessel 2][tessel-website].
+
+Naturally, in the IoT context **security** gains more importance, and I attended two interesting ~~advertisements~~ talks on [automated code vulnerability scanning][jacks-talk] with [Jacks][jacks-website] and [protecting your client-side JavaScript][jscrambler-talk] with [Jscrambler][jscrambler-website]. Both products provide convenient ways for developers to secure their JavaScript applications, withouth requiring an expertise in the domain of web application security.
+
+## 2. React still has huge momentum
+
+Just like the other conferences I attended in the last 12 months, _FullStack_ featured the canonical talk by a company that [moved from a monolithic [add your current technology here] application to a micro-service based system with a front-end powered by an isomorphic React & Redux app running on NodeJS][react-talk]
+
+In total, there were around half a dozen talks around the **NodeJS/React/Redux** stack that my team is using for building ePages' next generation frontend and website editor. One of the most interesting ones was by Forbes Lindesay from Facebook, who talked about **data fetching in React**. All in all there was plenty of opportunity to compare notes on that topic with other attendants from the very international audience.
+
+## 3. This doesn't mean framework fragmentation is a thing of the past
+
+{% imagecaption blog/blog-varnish.png 30% %}
+Can you spot the syntax error in the conference shirt?
+{% endimagecaption %}
+
+Of course there's enough space for other frameworks in the JavaScript world. **Angular2** was a big topic at _FullStack_, and after giving it a try in an intense workshop, I was left with the impression that the parts I liked most about it, namely **RxJS and TypeScript**, are by Microsoft. But it's good to see that Angular2 adds server side rendering or **Universal JavaScript**, and **Ember** is getting that as well, as pointed out in a keynote by Ember co-creator **Yehuda Katz**.
```

Comment:
Link the highlighted keywords to the respective websites, if available.
e.g.http://emberjs.com/